src/App.js

```diff
@@ -44,6 +50,29 @@ const NavBarPadding = Styled.div`
   }
 `;
 
+const Canvas = Styled.div`
+width:100%;
+height:200px;
+position: relative;
+margin-bottom:20px;
+img{
+  height: 100%;
+  width: 100%;
+  display: block;
+  margin: auto;
+  border-radius:5px;
+}
+h1{
+  text-align: center;
+  color:white;
+  position: absolute
+  top:50%;
+  left:50%;
+  transform: translate(-50%,50%);
```

Comment:
To center text simply change transform:translate to (-50%,-50%) I think the text sort of fades into the canvas however but can be fixed by changing font-color maybe?